matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/SendGossipRequestWorker.kt

```diff
@@ -58,7 +61,8 @@ internal class SendGossipRequestWorker(context: Context,
     }
 
     override suspend fun doSafeWork(params: Params): Result {
-        val localId = LocalEcho.createLocalEchoId()
+        // (temporary code)
```

Comment:
Temporary how/why?

Reply:
params.txnId should be provided in all cases now. But `Params` can be deserialized by the WorkManager from data serialized in a previous version of the application, so without the `txnId`. That's why it has to stay nullable for a while... Probably until we delete all this code to switch to the RustSdk.

Reply:
The comment is in this state isn't that helpful if that's going to hang around until the rust-sdk comes around. I would either expand it a bit to explain the state of affairs or just remove it to not confuse people that might be reading this code.

Reply:
You are right, I've updated the comment (at the 3 places).